pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -987,8 +987,9 @@ func registerCDC(r registry.Registry) {
 				opts:       map[string]string{"initial_scan": "'no'"},
 			})
 			ct.runFeedLatencyVerifier(feed, latencyTargets{
-				initialScanLatency: 3 * time.Minute,
-				steadyLatency:      5 * time.Minute,
+				// Higher latencies are permitted due to retry backoff
+				initialScanLatency: 10 * time.Minute,
+				steadyLatency:      30 * time.Minute,
```

Comment:
would be nice to figure out how to change that default instead of making tests take longer?